src/Mailgun/Message/MessageBuilder.php

```diff
@@ -451,7 +451,10 @@ public function setClickTracking($enabled)
      * @param string      $timeDate
      * @param string|null $timeZone
      *
-     * @return string
+     * @return MessageBuilder
+     *
+     * @deprecated Changed behaviour, returns reference to self for fluent interface.
```

Comment:
`@deprecated The return value is deprecated. This method will return $this in version 3.0.`